src/backend/d3d12/FramebufferD3D12.cpp

```diff
@@ -87,8 +79,7 @@ namespace d3d12 {
                 device->GetD3D12Device()->CreateRenderTargetView(device->GetCurrentTexture().Get(), &rtvDesc, rtvHandle);
             }
         }
-
-        if (usingBackbuffer) {
+        if (subpassInfo.depthStencilAttachmentSet) {
             args.dsv = dsvHeap.GetCPUHandle(0);
```

Comment:
ah, this is somewhat annoying. We have an index in subpassInfo but it's for all attachments, so it won't work for dsvHeap. I'll think about this.